src/views/Dashboard.vue

```diff
@@ -0,0 +1,29 @@
+<template lang="pug">
+  .dashboard-container
+    nav-bar
+</template>
+
+<script>
+import NavBar from './NavBar'
+
+export default {
+  name: 'dashboard',
+  components: {
+    NavBar
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/scss/commons/colors.scss';
+
+.dashboard-container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  background-color: $light-grey;
+  height: 100vh;
```

Comment:
No need for this

Reply:
Actually, I need it or the container will take the height of its children, in this case, the navbar. In the tree, above this container there's only one more div, the #app one. Should I apply this height to that div and then use a `height: 100%` in this one?

Reply:
It's OK for the container in this case to have the same height as the children, so leave it in `auto` :smile: